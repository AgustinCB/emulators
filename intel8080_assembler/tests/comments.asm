MVI A, 10
; This is a comment
;
MVI A, 10B