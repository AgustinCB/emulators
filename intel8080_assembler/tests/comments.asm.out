0000 MVI A,#$0a
0002 MVI A,#$02
